src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1062,6 +1070,14 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='Generate and assign an Azure Active Directory Identity for this server'
                    'for use with key management services like Azure KeyVault.')
 
+        c.argument('public_network_access',
```

Comment:
Since this is already defined in `sql server` scope, why do you need to repeat it here?